.gitignore

```diff
@@ -31,3 +31,6 @@ node_modules
 
 # Optional REPL history
 .node_repl_history
+
+# Npm Lock File
+package-lock.json
```

Comment:
We are disabling package-lock completely via `.npmrc`, see e.g. https://github.com/strongloop/loopback-next/blob/master/.npmrc

Please revert this change and add `.npmrc` file instead.